website/static/css/custom.css

```diff
@@ -37,6 +37,21 @@ a {
   color: $secondaryColor;
 }
 
+/* Change styling of selected navigation item */
+.navListItemActive {
+    display: flex;
+}
+
+.navListItemActive a {
+    color: red;
+}
+
+.navListItem.navListItemActive:before {
+  padding-left: 0.5rem;
+  padding-right: 0.25rem;
+  content: '🐐';
```

Comment:
:joy:

Let's go with 🐙 I think. That'd make more sense for everything I think.

Reply:
Fixed!